symtabAPI/CMakeLists.txt

```diff
@@ -98,7 +103,7 @@ dyninst_library(
   DEFINES SYMTAB_LIB
   DYNINST_DEPS common
   DYNINST_INTERNAL_DEPS dynDwarf dynElf
-  PRIVATE_DEPS OpenMP::OpenMP_CXX Dyninst::Boost Dyninst::ElfUtils
+  PRIVATE_DEPS OpenMP::OpenMP_CXX Dyninst::Boost Dyninst::ElfUtils Dyninst::PeParse
```

Comment:
@kupsch @bbiiggppiigg @wxrdnx @rochauha

This raises a point that I was thinking about the other day: should we extract PE parsing into its own library like dynElf? I think it might be useful. We can delve further into that once we get more PE parsing working rather than holding up @batuzovk's work . Ultimately, we don't want to do file parsing directly in the `Object` classes since that would create tight coupling between dyninstAPI and the third-party library (this is how we ended up with bad ROSE support in dataflowAPI).